### PR TITLE
fix: Double word typos

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
Based on the search:
`\bthe the\b|\bthat that\b|\bin in\b|\bis is\b|\band and\b|\bare are\b|\bon on\b|\bout out\b|\ba a\b|\bit it\b|\bor or\b|\bas as\b|\ban an\b`
in `*.java, *.html, Messages.properties`

- Typos fixed.
- CHANGELOGs Fix notes added.

## Related Issues
N/A

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [NA] Write tests
- [NA] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java [hunk 1]
```diff
@@ -943,7 +943,7 @@ && matchBodyPattern(msg1, errorPattern, sb)) {
             // HTTP param we are manipulating.
             // Note also that because this comment only needs to be added to the "SQL_OR_TRUE" and
             // not to the equivalent SQL_AND_FALSE, because of the nature of the OR
-            // and AND conditions in SQL.
+            // & AND conditions in SQL.
```
Comment (thc202) [inline addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java:946]: This is unnecessary.

File: addOns/bruteforce/src/main/javahelp/org/zaproxy/zap/extension/bruteforce/resources/help/contents/tab.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+l<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
```
Comment (thc202) [inline addOns/bruteforce/src/main/javahelp/org/zaproxy/zap/extension/bruteforce/resources/help/contents/tab.html:1]: Revert.

File: addOns/spiderAjax/CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Maintenance changes.
 
+### Fixed
+- A typo in the Options screen.
```
Comment (thc202) [inline addOns/spiderAjax/CHANGELOG.md:15]: API endpoint description.

File: addOns/wappalyzer/CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Fixed
+- A typos in the help content.
```
Comment (thc202) [inline addOns/wappalyzer/CHANGELOG.md:8]: `typo`

Comment (ricekot) [reply addOns/wappalyzer/CHANGELOG.md:8]: A typo in the word typo in a PR addressing typos 😆

Comment (kingthorin) [reply addOns/wappalyzer/CHANGELOG.md:8]: Hahahaha yeah 🤦‍♂️

**Comment by kingthorin** [comment]
Done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Comment by psiinon** [comment]
I wonder how many were my fault - probably most of them 😉

**Pull request merged**